src/components/Banner.vue

```diff
@@ -0,0 +1,208 @@
+<script setup>
+import { ref, computed } from "vue";
+import useMovieStore from "../store/movieStore";
+
+const movieStore = useMovieStore();
+
+const searchByTitle = ref(true);
+const query = ref('');
+const selectedMovie = computed(() => movieStore.getMovieById);
+
+const searchHandler = (option) => {
+  if (option === "title") searchByTitle.value = true;
+  else if (option === "genres") searchByTitle.value = false;
+  else searchByTitle.value = true;
+  movieStore.setFilterParam(option);
+};
+
+const clickHandler = () => {
+  movieStore.setSearchQuery(query.value);
+}
+
+</script>
+
+<template>
+  <div class="root">
+    <header>
+      <div class="header__left">
+        <span class="header__text1">netflix</span>
+        <span>roulette</span>
+      </div>
+      <div v-if="movieStore.isMovieDetailOpen" class="header__right">
+        <img src="../assets/search.svg" @click="movieStore.toggleMovieDetailPanel(null)">
+      </div>
+    </header>
+    <div v-if="movieStore.isMovieDetailOpen" class="banner">
+      <img class="banner-img"
+        :src="selectedMovie.posterurl"
+        :placeholder="selectedMovie.title" />
```

Comment:
The `img` tag doesn't have the `placeholder` attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img
Use `alt` instead